README.md

```diff
@@ -11,7 +11,16 @@
 
 ## Category 1
 
-* [Placeholder link]()
+* [Ubuntu HPC](https://www.youtube.com/watch?v=tGIobcyKViI&t=2s) - Introduce the cluster components in a clear arrangement and provide some resources to learn HPC clusters, as well as some use cases.
+* [AWS ParallelCluster login node](https://aws.amazon.com/cn/blogs/hpc/introducing-login-nodes-in-aws-parallelcluster/) - A detailed introduction and instruction to access the login node of AWS ParallelCluster, including the way to control the login nodes.
+* [RWTH High Performance Computing (Linux)](https://help.itc.rwth-aachen.de/en/service/rhr4fjjutttf/) - This instruction gives detailed description of each login node, including different login ways via SSH and MFA, remote desktop and Jupyter.
+* [NetApp ONTAP AI with NVIDIA DGX A100 Systems](https://www.netapp.com/media/19432-nva-1151-design.pdf) - This designing document gives detailed tables of the architecture of NetApp ONTAP AI and NVIDIA DGX A100 Systems, gives the topologies of compute nodes and storage hierarchy.
+* [Dell Technologies HPC solutions portfolio](https://www.delltechnologies.com/asset/en-ae/products/ready-solutions/briefs-summaries/ready-solutions-for-hpc-solution-overview.pdf) - This documents gives detailed configuration information of HPC. Not only the data, but also some interesting things like how the HPCs are configured, and not only CPUs and GPUs, FPGAs can also be used to configure an HPC.
+* [Vlaams Supercomputer Centrum VSC documentation](https://docs.vscentrum.be/index.html) - This document gives a detailed introduction of their storage infrastructure, especially the architecture of tier-1 and tier-2.
+* [Top 500 list](https://www.top500.org/) - This website gives the data of the top 500 listed supercomputers, which gives a scope of how fast and how much compute nodes are, and also an introduction and comparison among these computers.
+* [FRONTIER (ORNL’s exascale supercomputer)](https://www.olcf.ornl.gov/frontier/) - This page gives a scope of the top 1 supercomputer in 2023, and some data comparison.
+* [Aurora by Argonne National Laboratory](https://www.alcf.anl.gov/aurora) - This page gives the detailed data of the nodes in Aurora.
+* [Iowa State University High Performance Computing](https://www.hpc.iastate.edu/guides/introduction-to-hpc-clusters/what-is-an-hpc-cluster) - One abstract HPC topology is gotten here. This website also contains instructions of using some tools like SLURM.
```

Comment:
This is a great resource! Let's actually link to the first page in the guide and encourage users to read through the whole guide, because there is a ton of nice information here. Maybe:
```
* [Iowa State University HPC Guides](https://www.hpc.iastate.edu/guides/introduction-to-hpc-clusters) - A collection of introductory guides for HPC including SLURM, globus, unix, python, julia, and much more.
```